<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TlasOverviewPane</class>
 <widget class="QWidget" name="tlas_pane_">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>300</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="tlas_pane_vertical_layout_">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="tlas_pane_grid_layout_">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <layout class="QHBoxLayout">
       <item>
        <layout class="QVBoxLayout" name="column_0_vertical_layout_">
         <item>
          <layout class="QHBoxLayout" name="title_horizontal_layout_">
           <item>
            <widget class="ScaledPushButton" name="tlas_title_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>TLAS index 0</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontal_spacer_">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>5</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="column_0_horizontal_layout_">
           <item row="0" column="0">
            <layout class="QVBoxLayout">
             <item>
              <widget class="ScaledPushButton" name="tlas_title_address_">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>0x-</string>
               </property>
              </widget>
             </item>
             <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>5</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontal_spacer_1_">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>5</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QGridLayout" name="tlas_pane_grid_layout_1_">
             <item row="1" column="0">
              <widget class="ScaledLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>64</height>
                </size>
               </property>
               <property name="text">
                <string></string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="ScaledLabel" name="instance_count_">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>63</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignTop | Qt::AlignRight</set>
               </property>
               <property name="text">
                <string>- instances</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="ScaledLabel" name="blas_count_">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignTop | Qt::AlignRight</set>
               </property>
               <property name="text">
                <string>- BLASes</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="acceleration_structure_diagram_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>370</width>
               <height>220</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
               <kerning>false</kerning>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="tlas_pane_grid_layout_2_">
             <item row="0" column="0">
              <widget class="ScaledLabel" name="memory_spacing_">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string></string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="ScaledLabel" name="tlas_memory_size_">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>110</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignTop</set>
               </property>
               <property name="text">
                <string>TLAS (- MiB)</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="ScaledLabel" name="total_memory_size_">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignTop</set>
               </property>
               <property name="text">
                <string>Total (- MiB)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontal_spacer_2_">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>5</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="tlas_pane_line_separator_">
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>110</y>
           <width>61</width>
           <height>91</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::VLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout">
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout">
         <property name="topMargin">
          <number>40</number>
         </property>
         <property name="bottomMargin">
          <number>40</number>
         </property>
         <item row="0" column="0">
          <widget class="ScaledLabel" name="total_triangles_label_">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignTop</set>
           </property>
           <property name="rightMargin">
            <number>50</number>
           </property>
           <property name="text">
            <string>Total triangles:</string>
           </property>
          </widget>
         </item>

         <item row="0" column="1">
          <widget class="ScaledLabel" name="total_triangles_content_">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignTop</set>
           </property>
           <property name="rightMargin">
            <number>75</number>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>

         <item row="0" column="2">
          <widget class="ScaledLabel" name="unique_triangles_label_">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignTop</set>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="text">
            <string>Unique triangles:</string>
           </property>
          </widget>
         </item>

         <item row="0" column="3">
          <widget class="ScaledLabel" name="unique_triangles_content_">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignTop</set>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>

         <item row="1" column="0">
          <widget class="ScaledLabel" name="node_information_label_">
           <property name="text">
            <string>Node information</string>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>

         <item row="1" column="2">
          <widget class="ScaledLabel" name="build_flags_label_">
           <property name="text">
            <string>Build flags</string>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>

         <item row="2" column="0">
          <widget class="ScaledLabel" name="number_of_nodes_label_">
           <property name="text">
            <string>Number of nodes:</string>
           </property>
          </widget>
         </item>

         <item row="2" column="1">
          <widget class="ScaledLabel" name="number_of_nodes_content_">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>

         <item row="2" column="2">
          <widget class="ScaledLabel" name="allow_update_label_">
           <property name="text">
            <string>Allow update:</string>
           </property>
          </widget>
         </item>

         <item row="2" column="3">
          <widget class="ReadOnlyCheckBox" name="allow_update_content_">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>

         <item row="3" column="0">
          <widget class="ScaledLabel" name="number_of_box_nodes_label_">
           <property name="text">
            <string>Number of box nodes:</string>
           </property>
          </widget>
         </item>

         <item row="3" column="1">
          <widget class="ScaledLabel" name="number_of_box_nodes_content_">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>

         <item row="3" column="2">
          <widget class="ScaledLabel" name="allow_compaction_label_">
           <property name="text">
            <string>Allow compaction:</string>
           </property>
          </widget>
         </item>

         <item row="3" column="3">
          <widget class="ReadOnlyCheckBox" name="allow_compaction_content_">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>

         <item row="4" column="0">
          <widget class="ScaledLabel" name="number_of_instance_nodes_label_">
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="text">
            <string>Number of instance nodes:</string>
           </property>
          </widget>
         </item>

         <item row="4" column="1">
          <widget class="ScaledLabel" name="number_of_instance_nodes_content_">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         
         <item row="4" column="2">
          <widget class="ScaledLabel" name="low_memory_label_">
           <property name="text">
            <string>Low memory:</string>
           </property>
          </widget>
         </item>

         <item row="4" column="3">
          <widget class="ReadOnlyCheckBox" name="low_memory_content_">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>

         <item row="5" column="0">
          <widget class="ScaledLabel" name="number_of_inactive_instances_label_">
           <property name="text">
            <string>Number of inactive instances:</string>
           </property>
          </widget>
         </item>

         <item row="5" column="1">
          <widget class="ScaledLabel" name="number_of_inactive_instances_content_">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>

         <item row="5" column="2">
          <widget class="ScaledLabel" name="build_type_label_">
           <property name="text">
            <string>Build type:</string>
           </property>
          </widget>
         </item>

         <item row="5" column="3">
          <widget class="ScaledLabel" name="build_type_content_">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>

         <item row="6" column="0">
          <widget class="ScaledLabel" name="number_of_box16_nodes_label_">
           <property name="text">
            <string>Number of Box16 nodes:</string>
           </property>
          </widget>
         </item>

         <item row="6" column="1">
          <widget class="ScaledLabel" name="number_of_box16_nodes_content_">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>

         <item row="7" column="0">
          <widget class="ScaledLabel" name="number_of_box32_nodes_label_">
           <property name="text">
            <string>Number of Box32 nodes:</string>
           </property>
          </widget>
         </item>

         <item row="7" column="1">
          <widget class="ScaledLabel" name="number_of_box32_nodes_content_">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>

        </layout>
       </item>

       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScaledLabel</class>
   <extends>QLabel</extends>
   <header>qt_common/custom_widgets/scaled_label.h</header>
  </customwidget>
  <customwidget>
   <class>ScaledPushButton</class>
   <extends>QLabel</extends>
   <header>qt_common/custom_widgets/scaled_push_button.h</header>
  </customwidget>
  <customwidget>
   <class>ReadOnlyCheckBox</class>
   <extends>QCheckbox</extends>
   <header>views/custom_widgets/read_only_checkbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
